learningresources/search_indexes.py

```diff
@@ -0,0 +1,26 @@
+"""
+Index information for Haystack.
+
+http://django-haystack.readthedocs.org/en/latest/tutorial.html
+"""
+from haystack.indexes \
+    import SearchIndex, Indexable, CharField  # pragma: no cover
```

Comment:
I don't know, but the tests do use Haystack and it gets 0% coverage. I'll remove the `pragma` statements and check again to be sure, though.